modules/gapi/src/api/gmat.cpp

```diff
@@ -83,6 +83,9 @@ namespace{
 #if !defined(GAPI_STANDALONE)
```

Comment:
Just wondering, what is for the `GAPI_STANDALONE` case?
In this case, `cv::Mat` = `cv::gapi::own::Mat`.
Maybe this code could continue working in STANDALONE? (cc: @mpashchenkov )